src/repository/cert.rs

```diff
@@ -2210,22 +2609,16 @@ mod test {
     fn decode_and_inspect_certs() {
         Cert::decode(
             include_bytes!("../../test-data/ta.cer").as_ref()
-        ).unwrap().inspect_ta_at(
-            true, Time::utc(2020, 11, 1, 12, 0, 0)
-        ).unwrap();
+        ).unwrap().inspect_ta(true).unwrap();
```

Comment:
Why are the timestamps no longer passed in? Is the test still testing the same thing as a result?

Reply:
This has been moved from inspection to validation. This way, inspection can happen completely independent of outside factors, i.e., all you need for inspection is the certificate itself. The previous structure became problematic when composing errors.

Reply:
There now must be a call to `Cert::verify_validity` in all the `inspect_at` methods. I did check and these should be there, anyway, since the compiler would have complained about unused arguments otherwise, but if you want to double check, that’d be grand.